scripts/repl.js

```diff
@@ -204,6 +204,29 @@
     evt.stopPropagation();
   }
 
+  function envItemToString(item) {
+    return item.name + ' ' + JSON.stringify(item.targets);
+  }
+
+  function onEnvBuild(node, opts, envResult) {
```

Comment:
One thing to consider is, that once https://github.com/babel/babel-preset-env/pull/241 lands, we're likely going to follow that up with plugin-dependent debug output.

We might want to expose the debug output from directly from the plugin on `onPresetBuild` instead of needing to reconstruct it from structured data?

(And, maybe even stepping further back, @hzoo and I were discussing that maybe having debug output for _every_ transform would be really useful... so this might be the beginnings of a consistent way of accessing it?)